homeassistant/components/tuya/sensor.py

```diff
@@ -1238,7 +1238,7 @@ class TuyaSensorEntityDescription(SensorEntityDescription):
         ),
         TuyaSensorEntityDescription(
             key=DPCode.REVERSE_ENERGY_TOTAL,
-            translation_key="total_energy",
+            translation_key="total_production",
```

Comment:
Do you have a zndb fixture that can be used to show the changes?

Reply:
> > Do you have a zndb fixture that can be used to show the changes?
> 
> Yes, i will add a fixture.

Hi @zzysszzy did you find a fixture?

Reply:
I think it make better sense to use `total_energy_production`
```suggestion
            translation_key="total_energy_production",
```

What do you think @NoRi2909 ?

Reply:
Also, we do not have any fixtures using this currently, so it would be best to have a corresponding test fixture added (before this PR, or in this PR)

Reply:
A total is usually energy, while power is a momentary measurement.

But keeping "energy" in the name of the translation key can't hurt.